README.md

```diff
@@ -17,3 +17,5 @@ See [Douglas C. Schmidt website](https://www.dre.vanderbilt.edu/~schmidt)  for m
 networked applications, [TAO](https://www.dre.vanderbilt.edu/~schmidt/TAO.html) is a C++
 implementation of the OMG's [CORBA](https://www.omg.org/spec/CORBA/) standard. The main home page
 is [here](https://www.dre.vanderbilt.edu/~schmidt/).
+
+All support on github is **best effort**, when you require a guaranteed response consider hiring one of the [commercial companies](https://www.dre.vanderbilt.edu/~schmidt/commercial-support.html) who provide support and services for ACE/TAO.
```

Comment:
Some of the links on that vanderbilt.edu page are no longer valid.  Should we move the still-relevant parts of that page here?

Reply:
Is a file in git the best location? Doug can fix his page or maybe use the github wiki?

Reply:
I like the wiki idea so that project maintainers can update this info directly.

Reply:
Let us take that path, enable wiki and add a new page to which we can link from the readme

Reply:
I have added a new wiki page for commercial support, taken over the page from Doug with some cleanup, looks the links for OCI need to be updated @mitza-oci

Reply:
I updated the Wiki page.